config/apf/default/kustomization.yaml

```diff
@@ -0,0 +1,17 @@
+# This overlay builds the APF components to be used in combination
+# with other overlays.
+
+namespace: kueue-system
+namePrefix: kueue-
+
+labels:
+  - pairs:
+      app.kubernetes.io/name: kueue
+      app.kubernetes.io/component: controller
+    includeTemplates: true
+  - pairs:
+      control-plane: controller-manager
+    includeSelectors: true
```

Comment:
Could you check if we need to add the includeTemplates option?

Reply:
Ah, you right, we don't need both flags. We are not using selectors and templates on APF.